comm/src/main/scala/coop/rchain/comm/transport/GrpcTransportServer.scala

```diff
@@ -137,27 +137,6 @@ class TransportServer(server: GrpcTransportServer) {
 
   import CommunicationResponse._
 
-  def startWithEffects(
-      dispatch: Protocol => EitherT[Task, CommError, CommunicationResponse],
-      handleStreamed: Blob => EitherT[Task, CommError, Unit]
-  )(implicit log: Log[Task]): EitherT[Task, CommError, Unit] = {
-    val dis: Protocol => Task[CommunicationResponse] = msg =>
-      dispatch(msg).value.flatMap {
-        case Left(err) =>
-          Log[Task].error(
-            s"Error while handling message. Error: ${err.message}"
-          ) >> Task.now(notHandled(err))
-        case Right(m) => Task.now(m)
-      }
-    val hb: Blob => Task[Unit] = b =>
-      handleStreamed(b).value.flatMap {
-        case Left(err) =>
-          Log[Task].error(s"Error while handling streamed Packet message. Error: ${err.message}")
-        case Right(_) => Task.unit
-      }
-    EitherT.liftF(start(dis, hb))
-  }
```

Comment:
Please add `attemptAndLog` to `GrpcTransportServer.receive.dispatchInternal` in case `handleStreamed()` fails.
```scala
case msg: StreamMessage =>
  StreamHandler.restore(msg) >>= {
    case Left(ex) =>
      Log[Task].error("Could not restore data from file while handling stream", ex)
    case Right(blob) => handleStreamed(blob)
  }
```

Reply:
as discussed in call, this was only handling the `CommError` (not throwables) so nothing was removed really.